lib/components/mixins/tapable.dart

```diff
@@ -3,71 +3,78 @@ import 'dart:ui';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 
+import '../component.dart';
+import '../../extensions/offset.dart';
+import '../../extensions/vector2.dart';
 import '../../game/base_game.dart';
-import '../position_component.dart';
 
-mixin Tapable on PositionComponent {
-  void onTapCancel() {}
-  void onTapDown(TapDownDetails details) {}
-  void onTapUp(TapUpDetails details) {}
+mixin Tapable on Component {
```

Comment:
IMO we should scrap tappable and rethink the event propagation to the children. Just having a mixing for taps but ignoring all other gestures makes us go adapt our API to this specific case and possibly come up with something that would make it harder to detect other gestures.

Reply:
With the new `propagateToChildren` any gesture could in theory be handled.